cmd/vegawallet/commands/service_run.go

```diff
@@ -129,6 +129,11 @@ func BuildCmdRunService(w io.Writer, handler RunServiceHandler, rf *RootFlags) *
 		"",
 		"Path to the file containing the tokens database passphrase",
 	)
+	cmd.PersistentFlags().Int64Var(&f.TTL,
```

Comment:
`.Uint64Var()` would be better here.

Reply:
It is, for some reason it didn't show up in autocomplete, but looks like it's possible. Will change